add-on/chrome/modules/AboutLoop.jsm

```diff
@@ -77,38 +78,39 @@ var AboutLoop = {};
 // checks in mozilla-central (eg getUserMedia-related), so if we want to
 // make changes to the URL names themselves, we'll need to change them
 // there too...
-XPCOMUtils.defineLazyGetter(AboutLoop, "conversation", () => {
-  return new AboutPage("chrome://loop/content/panels/sidebar.html",
-                       "loopconversation",
-                       "E79DB45D-2D6D-48BE-B179-6A16C95E97BA",
-                       "About Loop Conversation",
-                       Ci.nsIAboutModule.ALLOW_SCRIPT |
-                        Ci.nsIAboutModule.HIDE_FROM_ABOUTABOUT |
-                        Ci.nsIAboutModule.MAKE_UNLINKABLE |
-                        Ci.nsIAboutModule.URI_SAFE_FOR_UNTRUSTED_CONTENT |
-                        Ci.nsIAboutModule.URI_MUST_LOAD_IN_CHILD);
-});
-
-XPCOMUtils.defineLazyGetter(AboutLoop, "panel", () => {
-  return new AboutPage("chrome://loop/content/panels/panel.html",
-                       "looppanel",
-                       "A5DE152B-DE58-42BC-A68C-33E00B17EC2C",
-                       "About Loop Panel",
-                       Ci.nsIAboutModule.ALLOW_SCRIPT |
-                        Ci.nsIAboutModule.HIDE_FROM_ABOUTABOUT |
-                        Ci.nsIAboutModule.MAKE_UNLINKABLE |
-                        Ci.nsIAboutModule.URI_SAFE_FOR_UNTRUSTED_CONTENT);
-});
+XPCOMUtils.defineLazyGetter(AboutLoop, "conversation", () =>
+  new AboutPage("chrome://loop/content/panels/sidebar.html",
+                "loopconversation",
+                "E79DB45D-2D6D-48BE-B179-6A16C95E97BA",
+                "About Loop Conversation",
+                Ci.nsIAboutModule.ALLOW_SCRIPT |
+                Ci.nsIAboutModule.HIDE_FROM_ABOUTABOUT |
+                Ci.nsIAboutModule.MAKE_UNLINKABLE |
+                Ci.nsIAboutModule.URI_SAFE_FOR_UNTRUSTED_CONTENT |
+                Ci.nsIAboutModule.URI_MUST_LOAD_IN_CHILD)
+);
+
+XPCOMUtils.defineLazyGetter(AboutLoop, "panel", () =>
+  new AboutPage("chrome://loop/content/panels/panel.html",
+                "looppanel",
+                "A5DE152B-DE58-42BC-A68C-33E00B17EC2C",
+                "About Loop Panel",
+                Ci.nsIAboutModule.ALLOW_SCRIPT |
+                Ci.nsIAboutModule.HIDE_FROM_ABOUTABOUT |
+                Ci.nsIAboutModule.MAKE_UNLINKABLE |
+                Ci.nsIAboutModule.URI_SAFE_FOR_UNTRUSTED_CONTENT)
+);
 
 XPCOMUtils.defineLazyGetter(AboutLoop, "toc", () => {
   return new AboutPage("chrome://loop/content/panels/toc.html",
```

Comment:
I think you might as well change the style of this one to match the other two that you've changed.